odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -660,6 +660,9 @@ def value_to_html(self, value, options):
         if not value:
             return ''
 
+        if not isinstance(value, models.Model) or not value._name == 'res.partner':
+            raise TypeError(_("The contact widget is only allowed on a partner field."))
```

Comment:
Why would a `TypeError` (or any error not inheriting from `UserError`) be translated?
It is meant for the dev, not for the user and Odoo is developed in English

Reply:
> Why would a `TypeError` (or any error not inheriting from `UserError`) be translated? It is meant for the dev, not for the user and Odoo is developed in English

Good point, this should either be an untranslated `TypeError` (for a very "raw" reporting I would expect, possibly even just a log error), or a translated `UserError`.